deploy/clowdapp.yaml

```diff
@@ -516,6 +518,9 @@ parameters:
 - name: NEWER_RELEASEVER_CSAF
   description: Enable finding updates/cves in newer releaseversion from CSAF
   value: "true"
+- name: VMAAS_VERSION_FILE_PATH
+  description: Path to the VMaaS VERSION file
+  value: "../../../VERSION"
```

Comment:
It does not seem correct, please use the absolute path. Based on the Dockerfile change it should be `/vmaas/VERSION`